.github/workflows/c.yaml

```diff
@@ -59,13 +61,16 @@ jobs:
                 jpeg-turbo \
                 libtiff \
                 openjpeg \
+                jxrlib \
                 gdk-pixbuf \
                 libxml2 \
                 sqlite \
                 cairo \
                 glib \
                 xdelta \
                 doxygen
+            # jxrlib from homebrew has a bug
+            LC_ALL=C sed -i '' 's/#endif UNREFERENCED_PARAMETER/#endif/' /usr/local/Cellar/jxrlib/1.1_1/include/jxrlib/JXRMeta.h
```

Comment:
Hmm, why is `LC_ALL=C` required here?

Reply:
There is a "illegal byte sequence" error in that sed command. LC_ALL=C fixes it.